dashboard-widgets.el

```diff
@@ -207,6 +207,16 @@ If nil it is disabled.  Possible values for list-type are:
   :type 'float
   :group 'dashboard)
 
+(defcustom dashboard-file-icon-height 1.0
```

Comment:
I suggest `dashboard-icon-file-height` because is a custom of `dashboard-icon` family ¿?